### ISSUE TITLE
Version the REST API

### ISSUE THREAD

**Issue by gz-c** [skycoin/skycoin#485]
API endpoints need to be prefixed with something like `/v1/` so we can handle breaking changes.
Need to figure out plan for how to do this correctly

**Comment by olemis** [comment]
Will this support different versions running in parallel (e.g. support `vx.y` for a while before deprecating it in favor of `v(x+1).y` so that implementors have time to migrate)

**Comment by gz-c** [comment]
yes

**Comment by olemis** [comment]
@gz-c is this a duplicate of #1444 ?
